src/integrationTest/java/uk/gov/hmcts/divorce/solicitor/SolicitorCreateTest.java

```diff
@@ -116,13 +117,13 @@ private String expectedResponse(final String jsonFile) throws IOException {
         return new String(readAllBytes(issueFeesResponseJsonFile.toPath()));
     }
 
-    private CaseData caseData() {
-        return CaseData.builder()
-            .petitionerFirstName(TEST_FIRST_NAME)
-            .petitionerLastName(TEST_LAST_NAME)
-            .petitionerEmail(TEST_USER_EMAIL)
-            .divorceOrDissolution(DivorceOrDissolution.DIVORCE)
-            .divorceCostsClaim(YES)
-            .build();
+    private Map<String, Object> caseData() {
+        Map<String, Object> caseData = new HashMap<>();
+        caseData.put("petitionerFirstName", TEST_FIRST_NAME);
+        caseData.put("petitionerLastName", TEST_LAST_NAME);
+        caseData.put("petitionerEmail", TEST_USER_EMAIL);
+        caseData.put("divorceOrDissolution", DivorceOrDissolution.DIVORCE);
+        caseData.put("divorceCostsClaim", YES);
```

Comment:
there is `CaseFieldsConstants` class which you can use to define these constants(and through out where you are using keys of case data)

Reply:
👍

Reply:
Have actioned this, thanks for the review @nitinprabhu